lib/js/belt_Id.js

```diff
@@ -2,12 +2,48 @@
 
 var Curry = require("./curry.js");
 
+function MakeComparableU(M) {
+  var cmp = M[/* cmp */0];
+  return /* module */[/* cmp */cmp];
+}
+
+function MakeComparable(M) {
+  var cmp = function (a, b) {
+    return Curry._2(M[/* cmp */0], a, b);
+  };
+  return /* module */[/* cmp */cmp];
+}
+
 function comparableU(cmp) {
   return /* module */[/* cmp */cmp];
 }
 
 function comparable(cmp) {
-  return comparableU(Curry.__2(cmp));
+  var M = /* module */[/* cmp */Curry.__2(cmp)];
+  var cmp$1 = M[/* cmp */0];
+  return /* module */[/* cmp */cmp$1];
```

Comment:
can you inline comparableU manually? it is a little verbose, but it is a one time effort which is worthwhile